src/parameterStore.ts

```diff
@@ -0,0 +1,13 @@
+import AWS from 'aws-sdk';
+
+const parameterStore = new AWS.SSM();
+
+export default async function getParameter(parameterName: string): Promise<string> {
+    const result = await parameterStore
+        .getParameter({
+            Name: parameterName,
+        })
+        .promise();
```

Comment:
I think we are going to want to handle the error where the parameter does not exist. In that case should it fail the deployment? I am thinking it should not so that we can deploy the solution without it. Otherwise we will need to provide the --issuerEndpoint option on initial fhirworks setup, where the parameter hasn't been created.

Reply:
I was wrong. This code will be running during any API request, and NOT during deployment. So the deployment would be always successful.

Reply:
I have added logs if any error happens here

Reply:
If errors happen here, API response would be 500

Reply:
Ah. All API requests will trigger a call to getFhirConfig()?

Reply:
Yes, that's one of the reason why Lambda is slow. It has to run the code from scratch every time.

Reply:
Hmm, can we confirm that the parameter store value is fetched only once and stored in the execution context? We don't want to have to fetch it on every API request.

Reply:
I will confirm that

Reply:
I confirmed that It didn't fetch the value on every API  request.